src/app/components/source/UserInfoEdit.js

```diff
@@ -553,7 +559,10 @@ class UserInfoEdit extends React.Component {
                   color="primary"
                   onClick={this.handleAddLink.bind(this)}
                 >
-                  {this.props.intl.formatMessage(messages.addLink)}
+                  <FormattedMessage 
+                    id={messages.addLink.id} 
+                    defaultMessage={messages.addLink.defaultMessage} 
```

Comment:
The desired usage is that the actual content of these keys is moved here, and `addLink` is to be removed from the `messages` object.